mlrun/api/alembic/tests/test_basic_migrations.py

```diff
@@ -0,0 +1,120 @@
+import logging
+
+import pytest
+from alembic.autogenerate.api import AutogenContext
+from alembic.autogenerate.render import _render_cmd_body
+
+from pytest_alembic.plugin.error import AlembicTestFailure
+
+log = logging.getLogger(__name__)
+
+
+# override default fixture for setting up propper config.
+@pytest.fixture
+def alembic_config():
+    return {"script_location": "alembic"}
+
+
+# Overridden Tests from Pytest-Alembic
```

Comment:
Are you sure all of this must be copy pasted from the `pyteste-alembic` package ? can't that be used from there somehow ? I see that those test are sitting inside the package code, maybe this is intentional so all you need to do is just import them here and they will run

Reply:
Not 100% sure. I definitely tried that. But couldn’t get it to work. Because as you see I needed to override the alembic_config fixture in order to run the tests with our config. And I’m not sure it’s possible without copying the tests over. But I’ll give it another shot and let you know.

Reply:
If the dir is named "migrations" instead of "alembic", then it can work without copying the tests. Cos that is the hardcoded default in pytest-alembic...
Changed the dir, a bit annoying but indeed cleaner.